client/my-sites/promote-post-i2/utils/index.ts

```diff
@@ -125,6 +125,22 @@ export const getCampaignDurationFormatted = ( start_date?: string, end_date?: st
 	return durationFormatted;
 };
 
+export const getCampaignActiveDays = ( start_date: string, end_date: string, duration: number ) => {
+	const dateStart = new Date( start_date );
```

Comment:
Maybe we can use moment here. That library make it easy to calculate date differences, and also make it easier for the reader to follow the code.